PSI Food waste/Pages/Forms/RestaurantVerified.cshtml.cs

```diff
@@ -55,7 +43,7 @@ public void OnGet()
         {
             ViewData["User"] = HttpContext.Session.GetString(key: "username");
             _productRepository.SortProducts();
-            products = _productRepository.GetList(_restaurantRepository.GetNextID());
+            products = _productRepository.GetList(_restaurantRepository.GetID());
 
```

Comment:
Yes, i was about to mention that GetNextId doesnt exist.

Reply:
mhm, I changed the name to make more sense